automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
@jyejare 

In this case I would do: `cv_name, _ = hammer_determine_cv_and_env_from_ak(activation_key, '1')` or `_, env_nam = hammer_determine_cv_and_env_from_ak(activation_key, '1')` depending on what var I want. But we can go with dict if you prefer ;)